components/cas-inbound-authenticator/src/main/java/org/wso2/carbon/identity/sso/cas/util/CASSSOUtil.java

```diff
@@ -242,7 +242,8 @@ public static TicketGrantingTicket getTicketGrantingTicket(String ticketGranting
             throws IdentityException {
         try {
             String username = String.valueOf(result.getSubject());
-            List<String> requestedClaims = new ArrayList<String>();
+            List<String> allClaims = new ArrayList<String>();
+            Map<String, String> claimsMap = new HashMap<String, String>();
```

Comment:
Shall we introduce a meaningful name here ? Claims map isn't very meaningful in this context. What claims do include in this map ?